packages/clerk_auth/lib/src/clerk_auth/auth.dart

```diff
@@ -35,16 +36,26 @@ class Auth {
           pollMode: pollMode,
         );
 
+  /// The injected Clerk publishable key
+  final String publishableKey;
+
+  /// Whether or not to send telemetry data (default true)
+  final bool telemetry;
+
   final Api _api;
 
   static const _codeLength = 6;
 
-  /// JsVersion of API
-  static const jsVersion = '4.70.0';
-
   /// default redirect URL for use with oAuth
   static const oauthRedirect = 'https://www.clerk.com/oauth-redirect';
 
+  /// Are we in development mode?
+  bool get isDevMode => env.display.instanceEnvironmentType.isDevelopment;
+
+  static const _telemetryPeriod = Duration(seconds: 27);
+  static final _telemetricEvents = <TelemetricEvent>[];
```

Comment:
Lets rename this to telemetricEventsQueue

Reply:
Let's also persist these with the Persistor. We probably want use an upper limit of 2000 events, treated as a FIFO queue. When adding an event the last events are popped out and dropped. Thus removing oldest entries.